app/src/main/res/layout/fragment_login.xml

```diff
@@ -1,92 +1,106 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools">
 
     <data></data>
-<ScrollView
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:fillViewport="true">
-    <LinearLayout
+
+    <ScrollView
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:background="#fff"
-        android:gravity="center"
-        android:orientation="vertical">
-
-        <ImageView
-            android:layout_width="120dp"
-            android:layout_height="120dp"
-            android:src="@drawable/logo_without_name" />
+        android:fillViewport="true">
 
-        <LinearLayout
-            android:id="@+id/lt_email"
+        <androidx.constraintlayout.widget.ConstraintLayout
             android:layout_width="match_parent"
-            android:layout_height="50dp"
-            android:layout_marginHorizontal="20dp"
-            android:layout_marginTop="20dp"
-            android:background="@drawable/stroke_rosasy"
-            android:orientation="horizontal"
-            android:padding="10dp">
+            android:layout_height="match_parent">
 
-            <EditText
-                android:id="@+id/et_mail"
-                android:layout_width="0dp"
-                android:layout_height="wrap_content"
-                android:layout_marginHorizontal="5dp"
-                android:layout_weight="1"
-                android:background="@android:color/transparent"
-                android:hint="Email"
-                android:imeOptions="actionNext"
-                android:inputType="text"
-                android:textAppearance="@style/TextAppearance.AppCompat.Medium" />
-        </LinearLayout>
+            <ImageView
+                android:id="@+id/img_logo"
+                android:layout_width="120dp"
+                android:layout_height="120dp"
+                android:src="@drawable/logo_without_name"
+                app:layout_constrainedWidth="false"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintLeft_toLeftOf="parent"
```

Comment:
Always, use start and end constraints instead of right and left this will cause an issue with RTL (Right to left) locales like Arabic language. 
So `app:layout_constraintLeft_toLeftOf="parent"` would be `app:layout_constraintStart_toStartOf="parent"` 

For the current constraints won't cause a big trouble because the left and right constraints are the same but when you want to set different  directional constraint would cause a problem as the left and right will be different based on the locale.